tap_shopify/streams/metafields.py

```diff
@@ -64,115 +60,140 @@ def get_query_params(self, updated_at_min, updated_at_max, cursor=None):
             params["after"] = cursor
         return params
 
-    def get_resource_type_query(self, resource):
+    @staticmethod
+    def get_resource_type_query(resource):
```

Comment:
this method is not utilised beyond the scope of this class and can be implemented as a Instance Method